### PR TITLE
Add CameraView and hook up with Mobilenet

### PR DESCRIPTION
**Pull request by yijianwang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "webpack --mode production",
     "start": "npm run build && node src/server/index.js",
-    "client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
```
[the remark this answers is not in the document]
Comment (yijianwang) [reply package.json]: it not needed, we already have devtool: "source-map" set up on webpack.config line 15. Also the auto rebuilding and refreshing are set in webpack.config 53.

File: webpack.config.js [hunk 2]
```diff
@@ -26,7 +26,7 @@ module.exports = {
       // TS loader
       {
         test: /\.tsx?$/,
-        use: "ts-loader",
```
[the remark this answers is not in the document]
Comment (yijianwang) [reply webpack.config.js]: I think awesome-typescript-loader is just a better/faster ts-loader (we are also using it in our SAC), I planned to replace the old ts-loader but forgot. https://www.npmjs.com/package/awesome-typescript-loader/v/3.0.3 (The best TypeScript loader for Webpack)

File: package.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MMoudy49) [reply package.json]: Oh okay. I'd like to learn more about it. Do you happen to have any documentation up that you can easily share with me? If not, I can definitely research myself but if you already have tabs open with the info, that's always easier. 😁

[the remark this answers is not in the document]
Comment (yijianwang) [reply package.json]: yes, for sure. There are a lot of good resources, for example the official doc talking about the configuration (dev tool) https://webpack.js.org/configuration/devtool/. My experience is to read other people's webpack config and use the offical doc/stack overflow/medium to understand line by line what each option is used for.

**Pull request merged**